### PR TITLE
Please check them and confirm then

### PR DESCRIPTION
**Pull request by halilarslan1999**

### CODE REVIEW

File: src/test/java/HalilPractice/HalilTest1.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package HalilPractice;
+
+public class HalilTest1 {
+    // Confirm it
+    //after check
```
Comment (UmitErsahis) [inline src/test/java/HalilPractice/HalilTest1.java]: buralari degistir

File: src/test/java/HalilPractice/HalilTest1.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package HalilPractice;
+
+public class HalilTest1 {
+    //please
```
Comment (Kkmyrat) [inline src/test/java/HalilPractice/HalilTest1.java:4]: it should be pls!

File: src/test/java/HalilPractice/HalilTest2.java [hunk 3]
```diff
@@ -0,0 +1,6 @@
+package HalilPractice;
+
+public class HalilTest2 {
+    //correct it
+    //after check
```
Comment (Kkmyrat) [inline src/test/java/HalilPractice/HalilTest2.java:5]: it should be check!!

**Review by Kkmyrat — APPROVED** [review]

File: src/test/java/HalilPractice/HalilTest1.java [hunk 4]
```diff
@@ -0,0 +1,5 @@
+package HalilPractice;
+
+public class HalilTest1 {
```
Comment (UmitErsahis) [inline src/test/java/HalilPractice/HalilTest1.java:3]: jhkjh

**Comment by UmitErsahis** [comment]
jbbk

File: src/test/java/HalilPractice/HalilTest1.java (hunk 1 above)
Comment (sumermehmet2001) [reply src/test/java/HalilPractice/HalilTest1.java]: gut gemacht

File: src/test/java/HalilPractice/HalilTest2.java (hunk 3 above)
Comment (sumermehmet2001) [reply src/test/java/HalilPractice/HalilTest2.java:5]: why to correct after check?

File: src/test/java/HalilPractice/HalilTest1.java (hunk 4 above)
Comment (sumermehmet2001) [reply src/test/java/HalilPractice/HalilTest1.java:3]: good practice

**Review by sumermehmet2001 — APPROVED** [review]

**Review by Ozleblebici — APPROVED** [review]

**Pull request merged**